airbyte-integrations/connectors/source-zendesk-support/acceptance-test-config.yml

```diff
@@ -31,6 +31,8 @@ acceptance_tests:
           # This stream is only available for enterprise accounts https://developer.zendesk.com/api-reference/ticketing/account-configuration/audit_logs/
           - name: "audit_logs"
             bypass_reason: "no records"
+          - name: "post_comments"
+            bypass_reason: "can not populate"
```

Comment:
why we can not populate this stream?

Reply:
posts (and comments for them) are available in zendesk community only. We have no subscription for this product yet.